python/fledge/services/core/support.py

```diff
@@ -123,6 +140,15 @@ def add_syslog_storage(self, pyz, file_spec):
             raise RuntimeError("Error in creating {}. Error-{}".format(temp_file, str(ex)))
         pyz.add(temp_file, arcname=basename(temp_file))
 
+    def add_syslog_service(self, pyz, file_spec, service):
+        # The fledge entries from the syslog file
+        temp_file = self._interim_file_path + "/" + "syslog-{}-{}".format(service, file_spec)
+        try:
+            subprocess.call("grep '{}' {} > {}".format(service, _SYSLOG_FILE, temp_file), shell=True)
```

Comment:
Filtered data is not upto the mark. It's free text search problem. See comments in FOGL-3826, syslog filtration asked in JIRA.